bin/index.js

```diff
@@ -105,6 +107,11 @@ const { open, port } = minimist(process.argv.slice(2), {
 });
 (async () => {
   try {
+    const { pkg } = await readPkgUp();
+    if (!pkg || !Object.keys(pkg).length) {
+      logError('This directory does not contain package.json file (via npm init).');
```

Comment:
Could we possibly reword this message to explain the problem a bit more?
Maybe something like this:
```suggestion
      logError('At the moment, Glicky does not support projects that have not been initialised with a package.json file. Please run `npm init` before running Glicky in this directory.');
```